magento/adobe-stock-integration
# Refactored redirect URI retrieving logic

### Description (*)
This PR refactors the comment model for system configuration. The new logic simplifies Redirect URI retrieval process and fixes an issue with incorrect redirect URI for some setups (check the original issue for details)

### Fixed Issues (if relevant)
1. #369 : Redirect URI and Redirect URI Pattern are not correct for some setups.

### Manual testing scenarios (*)

1. Go to Admin -> Stores -> Configuration -> System -> Adobe Stock Integration.
2. Copy the value from "Redirect URI:" comment.
3. Paste the URI in a new tab.
4. Make sure that a raw response is returned instead of 404 adminhtml page
